codalab/apps/web/static/js/worksheet/worksheet_interface.js

```diff
@@ -148,28 +152,26 @@ var Worksheet = React.createClass({
           $("#raw-textarea").focus();
         }
     },
-    toggleActionBar: function(){
-        this.setState({showActionBar:!this.state.showActionBar});
+    toggleActionBar: function() {
+        this.setState({showActionBar: !this.state.showActionBar});
     },
-    showActionBar: function(){
-        this.setState({showActionBar:true});
+    showActionBar: function() {
+        this.setState({showActionBar: true});
     },
-    hideActionBar: function(){
-        this.setState({showActionBar:false});
+    hideActionBar: function() {
+        this.setState({showActionBar: false});
     },
-    refreshWorksheet: function(){
+    refreshWorksheet: function() {
         $('#update_progress').show();
-        this.setState({updating:true});
+        this.setState({updating: true});
         ws_obj.fetch({
-            success: function(data){
-                // console.log("fetch_and_update success");
-                // console.log("%c--------------------------------------------------------------", "color: Green; font-size:15px;");
-                if(this.isMounted()){
+            success: function(data) {
+                if (this.isMounted()) {
                     this.refs.list.setState({worksheet: ws_obj.getState()});
```

Comment:
Yeah sure. I am using the following worksheet for testing. 

```
// Editing worksheet links(0xe87e3769a23947d38e58bdebbc28c068).
// https://github.com/codalab/codalab-cli/worksheet-syntax.md
//
[run run-echo-hello : echo hello]{0x3ba845bc288d466aa0d2e846b0242c95}
[dataset Pujun_Bhatnagar_Resume.pdf]{0x59103debd7744d8d899c5c666a0a82e8}
```

where the first item is a run while the second one is just a dataset. When I open the worksheet on the web-interface. The following is the activity of the console. (there were a bunch of warnings from React but I cleared the console to see what's exactly happening)

![00](https://cloud.githubusercontent.com/assets/5567951/10571032/a862b7a0-75ef-11e5-943d-e63658d4e0e2.png)
^ till now no error. I click on the first bundle and see the associated sub-files. 

![01](https://cloud.githubusercontent.com/assets/5567951/10571041/d4f9766e-75ef-11e5-9aa0-7589ed9d638a.png)
^ still no error and the stdout and stderr are shown in the side panel. 
However, when I click on the second bundle, which is a data set, I get an internal 500 error. 
![02](https://cloud.githubusercontent.com/assets/5567951/10571066/1a5467c8-75f0-11e5-859b-773f81442e32.png)
This is happening in line 231 in worksheet_sidepanel.js 
I uploaded another blank file `Pikachu.txt` as a dataset and that does not give this error.

![03](https://cloud.githubusercontent.com/assets/5567951/10571165/1cc22526-75f1-11e5-832a-d7dcdcaac3e7.png)
However, when I am trying to do the same with .pdf files, this error happens. 
![04](https://cloud.githubusercontent.com/assets/5567951/10571177/3bf38354-75f1-11e5-8e44-3caecbb01a78.png)
This is definitely odd. Might be related to pdf files. I was just playing around with this and came across this.